Cesium.Preprocessor/Cesium.Preprocessor.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>net7.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
Are you really sure you want to change only this?

Reply:
Some of the other projects are already migrated. I have no hesitations on upgrading the whole Cesium to net8 (except the runtime library which I still would like to target .NET Framework while possible).

Do you have some other suggestions?